test/integration/order_integration_test.rb

```diff
@@ -10,8 +10,9 @@ def setup
                         first_name: 'John',
                         last_name: 'Doe',
                         email: 'example@example.com')
-    @order = Order.create(total_cost: 100, user_id: user.id)
-    @item = order.items.create(title: 'Pour Over')
+    @item = Item.create(title: 'Pour Over', description: 'blah', price: 1000)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.